### PR TITLE
fixup translates

### PR DESCRIPTION
**Pull request by steve02081504**
There seems to be some problems:
- `data-i18n` is automatically trimmed on some browsers
- `data-i18n` content cannot have `\r` on some browsers

This fixes these and improves the Chinese translation btw

### CODE REVIEW

**Comment by steve02081504** [comment]
conflict resolved with https://github.com/SillyTavern/SillyTavern/pull/2384

File: .github/workflows/update-i18n.yaml [hunk 1]
```diff
@@ -20,7 +19,8 @@ jobs:
         aria2c https://raw.githubusercontent.com/SillyTavern/SillyTavern-i18n/main/generate.py
         aria2c https://raw.githubusercontent.com/SillyTavern/SillyTavern-i18n/main/requirements.txt
         pip install -r ./requirements.txt
-        python ./generate.py "" --sort-keys
+        python ./generate.py "" --sort-keys --auto-translate
```
Comment (Cohee1207) [inline .github/workflows/update-i18n.yaml]: Remove auto-translate, they come out in bad quality.

Comment (steve02081504) [reply .github/workflows/update-i18n.yaml]: fixed with [`d181635` (#2382)](https://github.com/SillyTavern/SillyTavern/pull/2382/commits/d181635460791c249b28557ebbd08752101647ee) and [`169f8f0` (#2382)](https://github.com/SillyTavern/SillyTavern/pull/2382/commits/169f8f0eb05ff2250bd89118bd7248f069a0e864)

File: public/scripts/i18n.js [hunk 2]
```diff
@@ -112,11 +112,17 @@ export function applyLocale(root = document) {
                 if (localizedValue || localizedValue == '') {
                     $(this).attr(attributeMatch[1], localizedValue);
                 }
+                else {
+                    console.warn(`Missing translation for '${attributeMatch[2]}'`);
```
Comment (Cohee1207) [inline public/scripts/i18n.js]: That is gonna cause a lot of console spam

Comment (steve02081504) [reply public/scripts/i18n.js]: According to my experiments, there are not many warnings
I personally think this helps translators and developers figure out the problem, what do you think?

Comment (steve02081504) [reply public/scripts/i18n.js]: I think I can add it back anytime, so [`822bcc6` (#2382)](https://github.com/SillyTavern/SillyTavern/pull/2382/commits/822bcc696039fe540d39f55795fe9f65a4111858) fixed

Comment (Cohee1207) [reply public/scripts/i18n.js]: Similar tools are in the debug menu and i18n repo, so this is redundant

**Pull request merged**